tests/integration/components/user-skills-input-test.js

```diff
@@ -20,12 +19,8 @@ let mockStore = {
 };
 
 let mockUserSkillsService = {
-  findUserSkill() {
-    return K;
-  },
-  removeSkill() {
-    return K;
-  }
+  findUserSkill() {},
+  removeSkill() {}
```

Comment:
@JoshSmith https://github.com/code-corps/code-corps-ember/pull/943/commits/48db11320568d13f9dd77482dc378f43a2588ff2 should resolve it. Let me know if that looks good.